WordPress/Classes/ViewRelated/Stats/Period Stats/Countries/CountriesCell.swift

```diff
@@ -66,7 +75,7 @@ private extension CountriesCell {
     }
 
     func setSubtitleVisibility() {
-        subtitleStackView.layoutIfNeeded()
```

Comment:
The `subtitleStackView` seems to be used to display the "Country" and "Views" labels. I don't see any visual issues.

I'm a bit confused about why `layoutIfNeeded()` is called here. The following line uses `subtitleStackView.frame.height`, but `subtitleStackView.layoutIfNeeded()` only lays out the subviews – it doesn't affect the height. I tried removing `layoutIfNeeded()`, and it works well. The stack view doesn't need `setNeedsLayout()` to update its layout.

The same comment applies to `TopTotalsCell`. Nothing changes if you remove `subtitleStackView.setNeedsLayout()`, which is what I would expect given it uses auto layout.

Reply:
Good observation, I'll check. `layoutIfNeeded` was added in this https://github.com/wordpress-mobile/WordPress-iOS/pull/16757 PR to allow subtitle text size to be dynamic but maybe it can be avoided altogether.